Dockerfile

```diff
@@ -1,12 +1,8 @@
 FROM jujusolutions/jujubox:latest
+MAINTAINER Charles Butler <charles.butler@canonical.com>
 
-VOLUME "/home/ubuntu/.juju"
-VOLUME "/home/ubuntu/trusty"
-VOLUME "/home/ubuntu/builds"
-VOLUME "/home/ubuntu/layers"
-VOLUME "/home/ubuntu/interfaces"
-ADD install-review-tools.sh /install-review-tools.sh
-RUN /install-review-tools.sh
-# Override jujubox run.sh
-ADD run.sh /run.sh
-CMD /run.sh
+ENV LAYER_PATH=/home/ubuntu/charms/layers
+ENV INTERFACE_PATH=/home/ubuntu/charms/interfaces
+
```

Comment:
I was suggesting making it a volume so it is always there and my script can mount it on my host. Either to deploy charms built on the host, or to make sure built charms reside on the host when the container is destroyed.